### PR TITLE
Hamzeh say Hi @casey-collab

### PR DESCRIPTION
**Pull request by hamzhk1**
@casey-collab

### CODE REVIEW

File: arabic/hi.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+...السلام عليكم و رحمة الله و بركاته...
```
Comment (casey-collab) [inline arabic/hi.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline arabic/hi.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
I'm closing all but your most recent pull request. If you still want this one merged, please re-open it.

**Pull request closed without merging**